src/amplify-migration/amplifyMigrationScript.ts

```diff
@@ -0,0 +1,289 @@
+const simpleGit = require("simple-git");
+const fs = require("fs");
+const glob = require("glob");
+const path = require("path");
+
+const jsdom = require("jsdom");
+const { JSDOM } = jsdom;
+const {
+  createAmplifyBranches,
+  startReleaseJob,
+  createAmplifyApp,
+  readBuildSpec,
+} = require("./amplifyUtils");
+const BRANCH_NAME = "chore-amplify-migration-change-permalinks";
+const ORGANIZATION_NAME = "isomerpages";
+
+interface AmplifyAppInfo {
+  appId: string;
+  repoName: string;
+  name: string;
+  repoPath: string;
+}
+
+const listOfRepos: [string, string][] = [
+  ["repo-name", "Human Friendly Name"], // Modify this line when running the script
+];
+
+async function main() {
```

Comment:
Yup, I do intend it to be run sequentially, easier to reconcile if errors are thrown.
Hmm, correct me if I am wrong, but even when using `Promise.all()`, we would still need to await so that we can get access to their resolved values correct? Ie. even with `Promise.all()`, we would still need to mark `main()` as `async`?

Reply:
yea i think i missed out the later half of my comment, which was that using `Promise.all` allows it to be parallel. 

since this doesn't affect end users, i'm fine w this as-is